### PR TITLE
liquidation skeleton code

### PR DESCRIPTION
**Pull request by yz89**

### CODE REVIEW

**Comment by kaichaosun** [comment]
The liquidation-client in my mind is aimed to be used by end uses like dev team, liquidators, it may consists a few components:
* a service/api to provide all the underwater account, so that end users can view and make the liquidate request in app.
* a cli/cron job to run by liquidators with the expected currencies.

**Comment by yz89** [comment]
> The liquidation-client in my mind is aimed to be used by end uses like dev team, liquidators, it may consists a few components:
> 
> * a service/api to provide all the underwater account, so that end users can view and make the liquidate request in app.
> * a cli/cron job to run by liquidators, which get the underwater account from service and make the liquidation with the provided currencies.

Good, let me raise another issue to track the advice.

File: config/decimal.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (GopherJ) [inline config/decimal.json:1]: it's a pain to maintain this in every place:)

File: src/index.ts [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+    console.log('borrower', param.borrower.toHuman());
+    console.log('liquidateToken', param.liquidateToken.toHuman());
+    console.log('collateralToken', param.collateralToken.toHuman());
+    console.log('repay', param.repay.toString());
+  });
+
+  await cryptoWaitReady();
+
+  const keyring = new Keyring({ type: 'sr25519' });
+  const alice = keyring.addFromUri('//Bob//stash', { name: 'Bob//stash default' });
```
Comment (GopherJ) [inline src/index.ts:184]: can we allow custom user's seed?

File: config/decimal.json (hunk 1 above)
Comment (yz89) [reply config/decimal.json:1]: I found it unnecessary to maintain and remove it.

**Comment by yz89** [comment]
Addressed comments in this commit https://github.com/parallel-finance/liquidation-client/commit/6eeb9bf7f0b4c2044ec7d4b48ef11aae1cf7d6be

**Pull request merged**